libfwupd/fwupd-enums.h

```diff
@@ -1168,6 +1168,14 @@ typedef enum {
 	 * Since: 2.0.0
 	 */
 	FWUPD_INSTALL_FLAG_NO_SEARCH = 1 << 8,
+	/**
+	 * FWUPD_INSTALL_FLAG_IGNORE_REQUIREMENTS:
+	 *
+	 * Ignore version requirement checks.
+	 *
+	 * Since: 1.9.21
+	 */
+	FWUPD_INSTALL_FLAG_IGNORE_REQUIREMENTS = 1 << 9,
```

Comment:
I wonder if you should re-order this above `FWUPD_INSTALL_FLAG_NO_SEARCH ` so that it backports more cleanly and we don't end up with a "hole"

Reply:
Hmm, NO_SEARCH should be `Since: 1.5.0` -- it's certainlt been in stable for some time. I'll fixup.